median/plugin.go

```diff
@@ -95,19 +109,25 @@ type latestRoundRequested struct {
 func (c *chainReaderContract) LatestTransmissionDetails(ctx context.Context) (configDigest ocrtypes.ConfigDigest, epoch uint32, round uint8, latestAnswer *big.Int, latestTimestamp time.Time, err error) {
 	var resp latestTransmissionDetailsResponse
 
-	err = c.chainReader.GetLatestValue(ctx, c.contract, "LatestTransmissionDetails", nil, &resp)
-	if err != nil {
+	err = c.chainReader.GetLatestValue(ctx, contractName, "LatestTransmissionDetails", nil, &resp)
+	if err != nil && !errors.Is(err, types.ErrNotFound) {
 		return
 	}
 
+	// Depending on if there is a LatestAnswer or not, and the implementation of the ChainReader,
+	// it's possible that this will be unset. The desired behaviour in that case is to have a zero value.
+	if resp.LatestAnswer == nil {
+		resp.LatestAnswer = new(big.Int)
+	}
+
 	return resp.ConfigDigest, resp.Epoch, resp.Round, resp.LatestAnswer, resp.LatestTimestamp, nil
 }
 
 func (c *chainReaderContract) LatestRoundRequested(ctx context.Context, lookback time.Duration) (configDigest ocrtypes.ConfigDigest, epoch uint32, round uint8, err error) {
 	var resp latestRoundRequested
 
-	err = c.chainReader.GetLatestValue(ctx, c.contract, "LatestRoundReported", map[string]any{"lookback": lookback}, &resp)
-	if err != nil {
+	err = c.chainReader.GetLatestValue(ctx, contractName, "LatestRoundRequested", nil, &resp)
```

Comment:
Just remembered that this is still LatestRoundReported in non evm relays `medianContract`

Reply:
We need a consistent name, we use the string LatestRoundRequested everywhere but here.  It can map to whatever name they use on-chain it's irrelevant.